moremesecons_entity_detector/init.lua

```diff
@@ -36,23 +37,24 @@ local object_detector_scan = function (pos)
 	local scanname = meta:get_string("scanname")
 	local scan_all = scanname == ""
 	local scan_names = scanname:split(',')
-	local radius = meta:get_int("radius")
-	if radius == 0 then
-		radius = 6
+	local radius = tonumber(meta:get("radius"))
+	local max_radius = moremesecons.setting("entity_detector", "max_radius", 16, 0)
+	if radius then
+		radius = math.min(radius, max_radius)
+	else
+		radius = math.min(6, max_radius)
```

Comment:
I think this can be shortened:
```
local max_radius = moremesecons.setting("entity_detector", "max_radius", 16, 0)
local radius = math.min(tonumber(meta:get("radius")) or 6, max_radius)
```